tests/versioning.go

```diff
@@ -3384,6 +3385,41 @@ func (s *VersioningIntegSuite) TestDescribeTaskQueueEnhanced_Unversioned() {
 
 }
 
+// TestDescribeTaskQueueEnhanced_GetBacklogInfoEmpty tests when there are no tasks in the backlog.
+// Tests when tasks are added and completed reside in workflow_task.go
```

Comment:
We also need a functional test that tests aggregation logic. i.e.:
1- add multiple tasks to multiple partitions and multiple build ids
2- verify that the returned backlog count matches the total number of tasks added for each build ID.

Reply:
as discussed async, shall take up the case covering multiple buildID's in the future.